lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -604,6 +617,12 @@ bool SessionStateEnforcer::set_upf_node(
   return true;
 }
 
+bool SessionStateEnforcer::inc_rtx_counter(

```

Comment:
No need to abbreviate here I think. 
`is_incremented_rtx_counter_within_max` maybe is more explicit

Reply:
renamed the function as you suggested